db/migrate/20190805204523_drop_deprecated_columns.rb

```diff
@@ -0,0 +1,47 @@
+class DropDeprecatedColumns < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :pipeline_run_stages, :failed_jobs
```

Comment:
@cdebourcy this blames to your PR #560. the col was last updated 2018-06-24.

Reply:
It's possible that the code path is broken. The column is "supposed" to be in use, so we would need to investigate that first

Reply:
see https://github.com/chanzuckerberg/idseq-web/blob/cd4311e3b3d1e42eacdc267c261f84331ad8cc56/app/models/pipeline_run_stage.rb#L145
and
https://github.com/chanzuckerberg/idseq-web/blob/cd4311e3b3d1e42eacdc267c261f84331ad8cc56/app/models/pipeline_run_stage.rb#L185

Reply:
Upon further thought, I think the reason this code path is not being used anymore is because we no longer is spot instances for the Batch jobs. Now we use on-demand instances. If for whatever reason (e.g. cost) we decided to go back to spot instances, this code would ensure that terminated Batch jobs get resubmitted (just the failed stage, unlike the more recent "retry from top"). So I'm in favor of keeping it

Reply:
Sounds good. I'll add the comment into the code.

Reply:
Thank you! I agree it's good to have that context in the code